src/web-rtc-client.js

```diff
@@ -671,25 +735,41 @@ export default class WebRTCClient extends Emitter {
     }
   }
 
-  _toggleMute(session: SIP.sessionDescriptionHandler, mute: boolean) {
+  _toggleMute(session: SIP.sessionDescriptionHandler, muteAudio: boolean) {
     const pc = session.sessionDescriptionHandler.peerConnection;
 
     if (pc.getSenders) {
       pc.getSenders().forEach(sender => {
         if (sender.track) {
           // eslint-disable-next-line
-          sender.track.enabled = !mute;
+          sender.track.enabled = !muteAudio;
         }
       });
     } else {
       pc.getLocalStreams().forEach(stream => {
         stream.getAudioTracks().forEach(track => {
           // eslint-disable-next-line
-          track.enabled = !mute;
+          track.enabled = !muteAudio;
         });
+      });
+    }
+  }
+
+  _toggleCamera(session: SIP.sessionDescriptionHandler, muteCamera: boolean) {
```

Comment:
Can be refactored with `_toggleMute`. And call them : `_toggleAudio` & `_toggleVideo`